server/app/serve_index.go

```diff
@@ -22,7 +22,7 @@ func serveIndex(ds model.DataStore, fs fs.FS) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		base := path.Join(conf.Server.BaseURL, consts.URLPathUI)
 		if r.URL.Path == base {
-			http.Redirect(w, r, base+"/", 302)
```

Comment:
A `302` isn't a `200` :/

Reply:
This is correct. GoLang weirdly names 302 as `http.StatusFound`. Ref: https://golang.org/pkg/net/http/

Reply:
It was originally 302, so I just replaced it with the appropriate constant. Should I do 200 instead?

Reply:
Nevermind, I thought that http.StatusFound was a 200, but it's indeed a [302](https://golang.org/pkg/net/http/)